### DISCUSSION TITLE
Scaling for a event with 350 users

### DISCUSSION THREAD

**Discussion by Clarens** [mozilla/hubs#5158] [Hubs Cloud Questions]
In a real-time event with around 350 active users that don't need to be all in the same room:

Is automated horizontal scaling supported?

Is automated vertical scaling supported?
 
How to scale the server using the same DNS?

Is it possible to create interconnected rooms on multiple servers? Sharing same scenes and even messages between them?
 
Why should we sign for a business server?
 
The best option is to have a single AWS instance more performant and a lot of users in few rooms or a lot of medium machines with a few users in many rooms?

**Comment by pattersonbl2** [comment]
It could be semi doable using a muti server enterprise setup.  This setup allows for separate voice and app servers and scaling horizontality for both.  You would need to keep room size under 25 people.  I am not sure about interconnection; it could cause an issue with performance.
